### DISCUSSION TITLE
List of third-party plugins?

### DISCUSSION THREAD

**Discussion by hellerbarde** [django-wiki/django-wiki#1156] [Q&A] [answered]
Is there a list/collection of plugins that have been written for django-wiki? Unfortunately, the project name doesn't lend itself to searching on pypi, so i have not been able to locate any so far. 

If you could point me to some plugins, I could write up a quick documentation page with links and descriptions.

**Comment by benjaoming** [comment, marked as answer]
There aren't so many, perhaps that's to blame on the lack of a good tutorial to encourage plugin writing.

See: https://github.com/search?q=django-wiki+plugin

> Unfortunately, the project name doesn't lend itself to searching on pypi

Yup, back in the days, `django-wiki` was taken and `wiki` was available.

**Comment by hellerbarde** [comment]
Understood. Thanks for the follow up! :)
